packages/rs-dpp/src/document/mod.rs

```diff
@@ -56,66 +74,87 @@ pub struct Document {
 }
 
 impl Document {
+    pub fn from_json_document(
+        raw_document: JsonValue,
```

Comment:
It's either raw or json - should it be called `json_dsocument` here?

Reply:
`json_document`  is correct. Good catch. Changed